src/backend/quarkus-management-api/src/main/java/com/nttdata/quarkus/management/api/service/ContactsService.java

```diff
@@ -0,0 +1,19 @@
+package com.nttdata.quarkus.management.api.service;
+
+import com.nttdata.quarkus.management.api.model.database.Contacts;
+
+import java.math.BigInteger;
+import java.util.List;
+
+
+public interface ContactsService {
+    List<Contacts> getContacts();
+
+    Contacts getContact(BigInteger contactId);
+
+    Contacts updateContact(Contacts contact);
+
+    Contacts addContact(Contacts contact);
```

Comment:
why are these returning `Contacts`? shouldnt they return a single Contact?

Reply:
The auto gen'd pluralized it because the database table name is plural.

Reply:
So it is actually a single "Contact"

Reply:
ahhh ok that's confusing haha

Reply:
but makes sense!

Reply:
Agreed